api/resources_portal/models/organization.py

```diff
@@ -33,10 +45,29 @@ class Meta:
     grants = models.ManyToManyField("Grant", through="GrantOrganizationAssociation")
 
 
+def assign_owner_perms(user, organization):
+    for permission in OWNER_PERMISSIONS:
+        assign_perm(permission[0], user, organization)
+
+
+def remove_owner_perms(user, organization):
+    for permission in OWNER_PERMISSIONS:
+        remove_perm(permission[0], user, organization)
+
+
+def assign_member_perms(user, organization):
+    for permission in MEMBER_PERMISSIONS:
+        assign_perm(permission[0], user, organization)
+
+
+def remove_member_perms(user, organization):
+    for permission in MEMBER_PERMISSIONS:
+        remove_perm(permission[0], user, organization)
```

Comment:
I love these functions, they're so nice and simple. However I think they'd be better as methods of the Organization class. Then they won't need the second parameter and they won't need to be imported separately.

Reply:
Smart. Just modified it to work this way.